src/adapters/ixVideo.js

```diff
@@ -0,0 +1,443 @@
+
+        // map request id to bid object to retrieve adUnit code in callback
+        bidRequests[bid.bidId] = {};
+        bidRequests[bid.bidId].prebid = bid;
+
+        let cygnusImpression = {};
+        cygnusImpression.id = bid.bidId;
+
+        cygnusImpression.ext = {};
+        cygnusImpression.ext.siteID = bid.params.video.siteID;
+        delete bid.params.video.siteID;
+
+        let podType = bid.params.video.startdelay;
+        if (bid.params.video.startdelay === 0) {
+          podType = 'preroll';
+        } else if (typeof START_DELAY_MAP[bid.params.video.startdelay] === 'undefined') {
+          podType = 'midroll';
+        }
+        cygnusImpression.ext.sid = [SLOT_ID_PREFIX_MAP[podType], 1, 1, 's'].join('_');
+
+        cygnusImpression.video = {};
+
+        if (bid.params.video) {
+          Object.keys(bid.params.video)
+            .filter(param => typeof VIDEO_REQUIRED_PARAMS_MAP[param] !== 'undefined' || typeof VIDEO_OPTIONAL_PARAMS_MAP[param] !== 'undefined')
+            .forEach(param => {
+              if (param === 'startdelay' && typeof START_DELAY_MAP[bid.params.video[param]] !== 'undefined') {
+                bid.params.video[param] = START_DELAY_MAP[bid.params.video[param]];
+              }
+              if (param === 'linearity' && typeof LINEARITY_MAP[bid.params.video[param]] !== 'undefined') {
+                bid.params.video[param] = LINEARITY_MAP[bid.params.video[param]];
+              }
+              cygnusImpression.video[param] = bid.params.video[param];
+            });
+        } else {
+          return;
+        }
+
+        let bidSize = getSizes(bid.sizes).shift();
+        if (!bidSize || !bidSize.width || !bidSize.height) {
+          return;
+        }
+        cygnusImpression.video.w = bidSize.width;
+        cygnusImpression.video.h = bidSize.height;
+
+        bidRequests[bid.bidId].cygnus = cygnusImpression;
+
+        return cygnusImpression;
+      });
+
+    let cygnusRequest = {
+      'id': bidRequest.bidderRequestId,
+      'imp': cygnusImpressions,
+      'site': {
+        'page': utils.getTopWindowUrl()
+      }
+    };
+
+    if (!utils.isEmpty(cygnusRequest.imp)) {
+      let cygnusRequestUrl = createCygnusRequest(cygnusRequest.imp[0].ext.siteID, cygnusRequest);
+
+      sendCygnusRequest(cygnusRequestUrl);
+    }
+  };
+
+  function createCygnusRequest(siteID, cygnusRequest) {
+    let cygnusUrl = (window.location.protocol === 'https:') ? url.parse(BASE_CYGNUS_VIDEO_URL_SECURE) : url.parse(BASE_CYGNUS_VIDEO_URL_INSECURE);
+    cygnusUrl.search.s = siteID;
+    cygnusUrl.search.r = encodeURIComponent(JSON.stringify(cygnusRequest));
+    let formattedCygnusUrl = url.format(cygnusUrl);
+    return formattedCygnusUrl;
+  }
+
+  function sendCygnusRequest(cygnusRequest) {
+    adloader.loadScript(cygnusRequest);
+  }
+
+  /* Notify Prebid of bid responses so bids can get in the auction */
+  window.handleCygnusResponse = function (response) {
```

Comment:
Please use the Prebid global object for handling your responses.
`$$PREBID_GLOBAL$$.handleCygnusResponse = function (response) {`
And then for your URL `const BASE_CYGNUS_VIDEO_URL_SECURE = 'https://as-sec.casalemedia.com/cygnus?v=8&fn='+encodeURIComponent($$PREBID_GLOBAL$$.handleCygnusResponse);

I presume this will work in your case but also prevent adding variables to the global scope.